django_loci/static/django-loci/js/loci.js

```diff
@@ -54,11 +54,11 @@ django.jQuery(function ($) {
     if (!gettext) { window.gettext = function (text) { return text; }; }
 
     function getLocationJsonUrl(pk) {
-        return baseLocationJsonUrl.replace('0000', pk);
+        return baseLocationJsonUrl.replace('00000000-0000-0000-0000-000000000000', pk);
```

Comment:
Why this change is needed? :thinking:

Reply:
Yes, the previous url regex was not very strict for UUID. Since, I rewrote the url patterns using `path`, correct placeholder are required to match the url pattern with uuid.